### DISCUSSION TITLE
sudo snap run rocketchat-server.initcaddy

### DISCUSSION THREAD

**Discussion by leisoneduardo** [RocketChat/rocketchat-server-snap#42] [General]
I'm following the steps to install rocket via snap [site ](https://docs.rocket.chat/quick-start/installing-and-updating/snaps/auto-ssl-with-snaps) but I found the error: sudo snap run rocketchat-server.initcaddy
error: cannot find app "initcaddy" in "rocketchat-server" and I can't go to the next steps to insert the ssl certificate.

Some help?

**Comment by debdutdeb** [comment]
The process changed a while ago, please check our updated documentation [here](https://docs.rocket.chat/quick-start/installing-and-updating/rapid-deployment-methods/snaps/auto-ssl-with-snaps#configure-everything-using-snap).
